Dockerfile.rocm

```diff
@@ -95,6 +98,17 @@ RUN if [ "$BUILD_CUPY" = "1" ]; then \
     && cd ..; \
     fi
 
+# build triton
+RUN if [ "$BUILD_TRITON" = "1"]; then \
+    mkdir -p libs \
+    && cd libs \
+    && pip uninstall -y triton \
+    && git clone https://github.com/ROCmSoftwarePlatform/triton.git
```

Comment:
Can you please try to build and verify this end to end? This line has error:
```
hongxyan@ctr-ubbsmc07:~/dockerx/ROCmVllm/vllm$ docker build -t triton_vllm_hongxia -f Dockerfile.rocm .
Sending build context to Docker daemon  9.722MB
Error response from daemon: dockerfile parse error line 107: unknown instruction: &&
```

Reply:
need to add the `\`to the end of the line here